ports/nrf/drivers/softpwm.h

```diff
@@ -1,3 +1,40 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2013-2016 The MicroPython-on-micro:bit Developers, as listed
+ * in the accompanying AUTHORS file
```

Comment:
I would have just written this as:

Copyright (c) 2013-2016 The MicroPython-on-micro:bit Developers: Damien P. George (@dpgeorge),
Nicholas H. Tollervey ...

Just to keep it compact, but still honest.